pkg/k8s/resource/resource.go

```diff
@@ -89,9 +89,15 @@ func FetchKubernetesResources(ctx context.Context, k *k8s.KubernetesAPI, options
 	}
 	resources = append(resources, clusterRoleBindings...)
 
-	roleBindings, err := fetchKubeSystemRoleBindings(ctx, k, options)
+	roles, err := fetchRoles(ctx, k, options)
 	if err != nil {
-		return nil, fmt.Errorf("could not fetch RoleBindings from kube-system namespace:%v", err)
+		return nil, fmt.Errorf("could not fetch Roles:%v", err)
+	}
+	resources = append(resources, roles...)
+
+	roleBindings, err := fetchRoleBindings(ctx, k, options)
+	if err != nil {
+		return nil, fmt.Errorf("could not fetch RoleBindings namespace:%v", err)
```

Comment:
nit: can we remove the namespace from the error message?

Reply:
oops, that somehow sneaked in from the last commit :(